.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -44,6 +44,7 @@ body:
         - area:gcp
         - area:gen-ai
         - area:graphql
+        - area:hardware
```

could we be consistent with the root namespace?
```suggestion
        - area:hw
```

(and in other places)

---

hw as abbreviation seems strange to me in registry so I prefer hardware, but I haven't changed the namespace.
@bertysentry do you think we should/can change common namespace for hardware to be indeed `hardware` and not `hw`?

---

@bertysentry could you please check this comment?

---

I'm not quite sure. I don't think the `area` field is supposed to match the namespace (see `gen-ai` which is not namespace for generative AI metrics).

So I think it's okay to keep `area:hardware` value, as it's just one of the possible options in the issue template.